### DISCUSSION TITLE
How can I obtain the forecast result with training set of data?

### DISCUSSION THREAD

**Discussion by Lucy-X-Chen** [winedarksea/AutoTS#64] [General]
I am able to get model result with model.results(), but how can I get the forecast result with training dataset? I would like to see the plot of actual vs forecast for the data used for training to.

Thank you!

**Comment by winedarksea** [comment]
This looks very similar to #61 and #62 

short answer, you can't easily write now (short of manually running AutoTS on the shortened data) but feel free to comment on #62 with ideas for how you'd want this to look when I add it.

**Reply by winedarksea** [reply]
One issue is multivariate series, this was designed for plotting hundreds of series, and that gets too much to fit on a graph @Lucy-X-Chen
